src/uct/ib/mlx5/gdaki/gdaki.c

```diff
@@ -34,17 +34,264 @@ ucs_config_field_t uct_rc_gdaki_iface_config_table[] = {
   {NULL}
 };
 
+
+static UCS_CLASS_INIT_FUNC(uct_rc_gdaki_ep_t, const uct_ep_params_t *params)
+{
+    uct_rc_gdaki_iface_t *iface = ucs_derived_of(params->iface,
+                                                 uct_rc_gdaki_iface_t);
+    uct_ib_mlx5_md_t *md = ucs_derived_of(iface->super.super.super.super.md,
+                                          uct_ib_mlx5_md_t);
+    uct_ib_iface_init_attr_t init_attr = {};
+    uct_ib_mlx5_qp_attr_t qp_attr      = {};
+    ucs_status_t status;
+
+    UCS_CLASS_CALL_SUPER_INIT(uct_base_ep_t, &iface->super.super.super.super);
+
+    init_attr.cq_len[UCT_IB_DIR_TX] = iface->super.super.config.tx_qp_len * 4;
```

Comment:
why CQ len should be *4 ?
shall be use MAX_BB constant?

Reply:
or take it from `init_attr` and make sure it is initialized in iface creation?